burn-core/src/optim/adam.rs

```diff
@@ -173,14 +170,16 @@ mod tests {
     use crate::tensor::{Data, Distribution, Tensor};
     use crate::{nn, TestADBackend, TestBackend};
 
+    static LEARNING_RATE: f64 = 0.01;
```

Comment:
Probably const is better since the value is used to initialize and not make it available globally. Also, for consistency, should use `LearningRate` instead of 64.